src/components/calcite-alert/calcite-alert.tsx

```diff
@@ -92,6 +92,19 @@ export class CalciteAlert {
     this.requestedIcon = setRequestedIcon(StatusIcons, this.icon, this.color);
   }
 
+  @Watch("autoDismissDuration")
+  updateDuration(newDuration: string, prevDuration: string): void {
+    if (this.autoDismiss && prevDuration !== newDuration) {
+      if (this.autoDismissTimeout) {
```

Comment:
Can you combine these if statements? For better legibility, we can avoid nested if statements.